README.rst

```diff
@@ -224,7 +282,7 @@ Executing custom commands
 -------------------------
 
 For experimenting it can be useful to execute arbitrary commands against the endpoint.
-You can access the available methods by calling ``songpal list-all``.
+You can access the available methods by calling ``songpal list_all``.
```

Comment:
Please revert this.

Reply:
Sorry, thought that was gone already, should be fixed now